client/src/components/Contact/Contact.js

```diff
@@ -1,62 +1,124 @@
-import React from "react";
+import React, { useState } from "react";
 import "../../css/Contact.css";
+import { set } from "mongoose";
 
 const Contact = () => {
+
+  const [newContact, setNewContact] = useState({
+    name: "",
+    email: "",
+    website: "",
+    comment: ""
+  })
+  
+
+  const handleContact = (e) => {
+    const addContact = {
+      ...newContact,
+      [e.target.name]: e.target.value,
+    };
+
+    setNewContact(addContact);
+  };
+
+
+  const handleSubmit = (e) => {
```

Comment:
I would rather put all the code handling add contact in one one function called createNewContact then call it inside handle submit.

also, handle sumbit should be called handleSubmitAddContactButton or similar name